.github/workflows/deployment-checks.yml

```diff
@@ -122,7 +132,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Check Mainnet Deployment Addresses
+      - name: Check Mainnet Action IDs
```

Comment:
Could do this in a separate PR, if we're being strict

Reply:
ah, thanks; I think it's fine.